README.md

```diff
@@ -83,19 +83,21 @@ This is an open source project managed by the Algorand Foundation. See the [cont
 
 ## Prerequisites
 
-The key required dependency is Python 3.10+, but some of the installation options below will install that for you.
-
-AlgoKit also has some runtime dependencies that also need to be available for particular commands.
+AlgoKit has some runtime dependencies that also need to be available for particular commands.
 
 > **Note**
-> You can still install and use AlgoKit without these dependencies and AlgoKit will tell you if you are missing one for a given command.
+> You can still install and use AlgoKit without these dependencies, and AlgoKit will tell you if you are missing one for a given command.
 
 - Git - Git is used when creating and updating projects from templates
 - Docker - Docker Compose (and by association, Docker) is used to run the AlgoKit LocalNet environment, we require Docker Compose 2.5.0+
 
+> **Note**
+> If you are using pipx to install AlgoKit, the key required dependency is Python 3.10+.
+
+
```

Comment:
Maybe we should add a note, saying if you have previously installed AlgoKit using pipx, however would like to switch to using one of the other installations method, please uninstall using `pipx uninstall algokit`, before installing using the instructions for your desired platform.